ui/components/app/snaps/snap-ui-tooltip/snap-ui-tooltip.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { FunctionComponent, ReactNode } from 'react';
+import Tooltip from '../../../ui/tooltip';
+
+export type SnapUITooltipProps = {
+  content: ReactNode;
+};
+
+export const SnapUITooltip: FunctionComponent<SnapUITooltipProps> = ({
+  content,
+  children,
+}) => {
+  return (
+    <Tooltip
+      html={content}
+      position={'bottom'}
```

Comment:
Do we always want to use `bottom` here? 🤔

Reply:
I found out that the tooltip has a logic to use `top` if the tooltip would fall out of the viewport :)

Reply:
I discussed with @eriknson and for the initial implementation we don't want to let the position being ruled by the snap. He wanted to have it dynamically set to `top`/`bottom` depending on the position of the element on the screen but this is hard to nail so I decided to keep it simple for now